pkg/systemstatsmonitor/cpu_collector.go

```diff
@@ -238,19 +252,6 @@ func (cc *cpuCollector) recordUsage() {
 }
 
 func (cc *cpuCollector) recordSystemStats() {
-	if cc.mSystemProcessesTotal == nil {
-		return
-	}
-	if cc.mSystemProcsRunning == nil {
-		return
-	}
-	if cc.mSystemProcsBlocked == nil {
-		return
-	}
-	if cc.mSystemInterruptsTotal == nil {
```

Comment:
will the cpu_collector break if the config is not applied?

Reply:
No - this is a nil check for the metric (as opposed to the config). The metric will only be initialized if the config is provided:

https://github.com/kubernetes/node-problem-detector/blob/1a7aa6505dfd72e6bffe7feb473e073194337b06/pkg/systemstatsmonitor/system_stats_monitor.go#L79-L81

So this will always evaluate to false.